dictionaries/git.json

```diff
@@ -6,7 +6,7 @@
 "TEUG": "tig",
 "TEUGS": "tig status",
 "TKPWEUT": "git",
-"AD/-P": "add .",
```

Comment:
Here, I'm building on the convention set up in the [code dictionary for `"-P": ".",`](https://github.com/didoesdigital/steno-dictionaries/blob/master/dictionaries/code.json#L67), and would prefer to keep that, as `PO*EUP` is the [code dictionary brief for `"{^}../{^}",`](https://github.com/didoesdigital/steno-dictionaries/blob/master/dictionaries/code.json#L94)

Reply:
Okay, understood. I'll remove the commit that adds this.